test/test_other.py

```diff
@@ -3761,9 +3742,8 @@ def test_emit_tsd_sync_compilation(self):
     shared.check_call(cmd)
 
   def test_emit_tsd_wasm_only(self):
-    err = self.expect_fail([EMCC, test_file('other/test_emit_tsd.c'),
-                            '--emit-tsd', 'test_emit_tsd_wasm_only.d.ts', '-o', 'out.wasm'])
-    self.assertContained('Wasm only output is not compatible --emit-tsd', err)
+    expected = 'Wasm only output is not compatible --emit-tsd'
```

Comment:
```suggestion
    expected = 'Wasm only output is not compatible with --emit-tsd'
```
pre-existing typo, might be worth fixing separately